pages/index.js

```diff
@@ -1,18 +1,52 @@
+import { nanoid } from "nanoid";
+import { useState } from "react";
+
 import Head from "next/head";
-import styled from "styled-components";
+import ShoppingList from "../components/ShoppingList";
 
 export default function Home() {
+  const [shoppingListItems, setShoppingListItems] = useState(shoppingListDB);
+
   return (
     <div>
       <Head>
-        <title>My App</title>
+        <title>MyShoppingManager</title>
         <meta name="description" content="Generated by create next app" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
 
       <main>
-        <h1>My App</h1>
+        <h1>MyShoppingManager</h1>
+        <ShoppingList items={shoppingListItems} />
       </main>
     </div>
   );
 }
+
+const shoppingListDB = [
```

Comment:
please use UPPERCASE for real constants

Reply:
I would turn the array into a "let" variable then, because it will be changed in an upcoming story of that sprint.

Reply:
You only write code for the moment, don't anticipate future changes. Oh and I didn't mean 'const', I meant real constants like inital data or api keys, to indicate that they never change you normally write them in all caps with lower dashes, i.e. "MONGODB_API_KEY"